lib/safira/store/store.ex

```diff
@@ -89,7 +89,7 @@ defmodule Safira.Store do
     |> Map.fetch!(:redeemables)
     |> Enum.map(fn redeemable ->
       b = get_keys_buy(attendee.id, redeemable.id)
-      Map.put(redeemable, :quantity, b.quantity)
+      Map.put(%{name: redeemable.name}, :quantity, b.quantity)
```

Comment:
This function `get_attendee_redeemables/1`, is used in AttendeeController, and its result is passed as an argument to the AttendeeControllerView, and then to RedeemableControllerView. So I don't think you should mess with its functionality. My suggestion would be to create a function for the desired purpose, or a mini function that uses this one and adds the step of changing the Map.

Reply:
I'll simply undo this changes. The task should work anyway